apps/core/src/config/local-cognito-jwt-verifier.ts

```diff
@@ -0,0 +1,35 @@
+import { CognitoJwtVerifier } from 'aws-jwt-verify';
+import { JwksCache, JwkWithKid, Jwks } from 'aws-jwt-verify/jwk';
+
+// JwksCache built for local development, works in conjunction with cognito-local
+const localJwksCache: JwksCache = {
+  getJwk: (): Promise<JwkWithKid> => Promise.resolve(localJwk),
+  getCachedJwk: (): JwkWithKid => localJwk,
+  addJwks: (): void => {
+    /** NOOP **/
+  },
+  getJwks: (): Promise<Jwks> => Promise.resolve(localJwks),
+};
```

Comment:
Nit: Do you need to annotate the return types of these functions? `JwksCache` doesn't add that for you?

Reply:
the return type declaration is not necessary, I used it as a helper to generate the properties during development here.
Are you suggesting we include only the necessary return type in general?